translate/storage/pocommon.py

```diff
@@ -196,18 +207,19 @@ def encodingToUse(encoding):
 
 class pofile(poheader.poheader, base.TranslationStore):
     Name = "Gettext PO file"  # pylint: disable=E0602
-    Mimetypes = ["text/x-gettext-catalog", "text/x-gettext-translation", "text/x-po", "text/x-pot"]
+    Mimetypes = ["text/x-gettext-catalog", "text/x-gettext-translation",
+                 "text/x-po", "text/x-pot"]
```

Comment:
From our styleguide: For lists or tuples with many items, break immediately after the opening brace

So:

```python
    Mimetypes = [
        "text/x-gettext-catalog",
        "text/x-gettext-translation",
        "text/x-po",
        "text/x-pot",
    ]
```